Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -13598,6 +13603,41 @@ - (void)_closeCurrentTypingCommand
     _page->closeCurrentTypingCommand();
 }
 
+- (UIView *)_selectionContainerViewAboveText
+{
+    // FIXME: Once -selectionContainerView (or an equivalent) is available as API, we should adopt that and
```

Comment:
is there a bugzilla bug or radar number you can associate with this?

Reply:
Not at the moment. I filed [280623](https://bugs.webkit.org/show_bug.cgi?id=280623) to track overall enablement, and [280624](https://bugs.webkit.org/show_bug.cgi?id=280624) to track adoption of the requisite API.